backend/fms_core/template_importer/_constants.py

```diff
@@ -16,6 +16,7 @@
 VALID_QC_FLAG_CHOICES = [PASSED, FAILED]
 # QC Instruments
 LIBRARY_QC_QUALITY_INSTRUMENTS = ["Caliper LabChip", "TapeStation"]
+SAMPLE_QC_QUALITY_INSTRUMENTS = ["Agarose Gel", "BioAnalyzer", "Caliper LabChip", "NanoDrop", "TapeStation", "Tecan Absorbance"]
```

Comment:
I think the order in the template was from most used to least used ... We might want to keep the order from the template. The fact that she mention TapeStation as missing while it is not (merely need to scroll) make me think the order is important to them.

Reply:
I think we should teach the lab the right way of looking for instruments. Going by importance is going to be annoying when we have different labs using freezeman.

Reply:
obviously, this task in itself was pretty menial, we have to be careful to not customize too heavily on what the users are suggesting (as ulysse mentioned, the solution was to simply to scroll down). this type of "task" seems to be a recurrent one. im on board with nafiz but i dont think this is where we should make that kind of decision.

im going to keep the order on the template but i would like to remark that i agree with nafiz but this would open up an entire discussion that we could address when gathered at the office.